contracts/plugins/assets/compoundv3/CusdcV3Wrapper.sol

```diff
@@ -286,9 +286,7 @@ contract CusdcV3Wrapper is ICusdcV3Wrapper, WrappedERC20, CometHelpers {
         (, uint64 trackingSupplyIndex) = getSupplyIndices();
         uint256 indexDelta = uint256(trackingSupplyIndex - baseTrackingIndex[account]);
 
-        baseTrackingAccrued[account] += safe64(
-            (safe104(accountBal) * indexDelta) / TRACKING_INDEX_SCALE
-        );
+        baseTrackingAccrued[account] += (safe104(accountBal) * indexDelta) / TRACKING_INDEX_SCALE;
```

Comment:
Is there a reason to not just use full precision here?

Reply:
What do you mean, full precision? The math here is in `uint256` and the product is before the division.

Reply:
Nvm, the comment is incorrect, I understood it wrong.